src/MusicalScore/ScoreIO/MusicSymbolModules/RepetitionCalculator.ts

```diff
@@ -42,6 +42,9 @@ export class RepetitionCalculator {
   }
 
   private handleRepetitionInstructions(currentRepetitionInstruction: RepetitionInstruction): boolean {
+    if (!this.currentMeasure) {
+      return false;
+    }
```

Comment:
we could instead handle this in calculateRepetitions:28, where the currentMeasure is set.